src/content/app/tools/blast/views/blast-form/BlastForm.scss

```diff
@@ -9,20 +9,19 @@
 
 .mainContainer {
   height: 100%;
-  padding: 0 30px 0 65px;
 }
 
 .mainContainerSmall {
   overflow: auto;
   scroll-behavior: smooth;
-  padding-bottom: $global-padding-bottom;
+  padding: var(--blast-page-standard-padding);
 }
 
 .grid {
   display: grid;
   grid-template-columns: repeat(2, min-content);
   column-gap: 45px;
-  padding-bottom: $global-padding-bottom;
+  padding: var(--blast-page-standard-padding);
```

Comment:
So here's the problem with the scrollbar that we observed. It suggests that our approach with the global BLAST-wide padding may be an incorrect one; because we may want to hand-adjust paddings for scrollable and not-scrollable containers.

_(or maybe that we need a `--blast-page-scrollable-padding` variable, besides  the `--blast-page-standard-padding` one)_


Uploading Screen Recording 2022-11-24 at 11.47.32.mov…

Reply:
@azangru this should be fixed with my last commit.